repository-data/webfiles/src/main/resources/site/freemarker/common/search-banner/search-alternative.ftl

```diff
@@ -0,0 +1,32 @@
+<#ftl output_format="HTML">
+<#include "../include/imports.ftl">
+
+<@hst.setBundle basename="website.labels"/>
+<@fmt.message key="labels.search-placeholder" var="placeholderText"/>
+<@fmt.message key="labels.search-button" var="buttonText"/>
+<@fmt.message key="labels.search-title" var="searchTitle"/>
+<@fmt.message key="labels.search-banner-text" var="searchBannerText"/>
+
+<#if facets??>
```

Comment:
already in first "include"

Reply:
@wojtek-oledzki Thanks for the heads up, I'll remove it.